### PR TITLE
Fix multiclass_nms_op unit test fail in python3.6

### PR DESCRIPTION
**Pull request by velconia**
According to this [issue](https://github.com/PaddlePaddle/Paddle/issues/12040), we should fix the unit test failures in python 3.6 and python 3.7  PR_CI

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py [hunk 1]
```diff
@@ -210,7 +215,7 @@ def test_check_output(self):
 class TestMulticlassNMSOpNoOutput(TestMulticlassNMSOp):
     def set_argument(self):
         # Here set 2.0 to test the case there is no outputs.
-        # In practical use, 0.0 < score_threshold < 1.0 
+        # In practical use, 0.0 < score_threshold < 1.0
```
Comment (junjun315) [inline python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py:218]: nothing change

File: python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py [hunk 2]
```diff
@@ -145,10 +145,15 @@ def batched_multiclass_nms(boxes, scores, background, score_threshold,
         lod.append(nmsed_num)
         if nmsed_num == 0: continue
 
+        tmp_det_out = []
         for c, indices in nmsed_outs.items():
             for idx in indices:
                 xmin, ymin, xmax, ymax = boxes[n][idx][:]
-                det_outs.append([c, scores[n][c][idx], xmin, ymin, xmax, ymax])
+                tmp_det_out.append(
```
Comment (junjun315) [inline python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py:152]: can add some notes. after py3.6, dict is orderly

**Review by junjun315 — APPROVED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py (hunk 2 above)
Comment (velconia) [reply python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py:152]: Thanks, I will add it later

File: python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py (hunk 1 above)
Comment (velconia) [reply python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py:218]: The whitespace in the end of line is stripped

**Pull request merged**